dev-qt/qtopengl/qtopengl-5.15.2.9999.ebuild

```diff
@@ -19,7 +19,6 @@ DEPEND="
 	=dev-qt/qtcore-${QT5_PV}*:5=
 	=dev-qt/qtgui-${QT5_PV}*[gles2-only=]
 	=dev-qt/qtwidgets-${QT5_PV}*[gles2-only=]
-	!gles2-only? ( virtual/opengl )
```

Comment:
the commit message should say `virtual/opengl` instead of `dev-qt/qtopengl`

Reply:
yup sorry about that, will make sure it is fine in gentoo.git.